assets/admin/package.json

```diff
@@ -9,6 +9,7 @@
         "watch": "webpack --mode development --watch"
     },
     "dependencies": {
+        "@ckeditor/ckeditor5-font": "^27.1.0",
```

Comment:
It is required to use the correct version of a plugin which matche Sulu CKEditor version: https://github.com/sulu/sulu-demo/pull/77#discussion_r527728655